src/NJsonSchema.CodeGeneration.Tests/EnumGenerationTests.cs

```diff
@@ -287,23 +287,23 @@ public async Task When_property_is_nullable_and_enum_allows_null_then_no_excepti
         {
             //// Arrange
             var json = @"{  
-���""type"":""object"",
-���""properties"":{  
-������""paataenktHandling"":{  
-���������""title"":""paataenktHandling"",
-���������""description"":""EAID_D38C4D27_B57C_4356_89E1_05E8DA0250B6"",
-���������""type"":[  
-������������""string"",
-������������""null""
-���������],
-���������""enum"":[  
-������������""�ndring"",
-������������""Nyoprettelse"",
-������������""Udg�r"",
-������������null
-���������]
-������}
-���}
+   ""type"":""object"",
```

Comment:
I don't understand the nature of that changes. It seems that something was wrong with encoding.